apps/kitchen-sink/src/ensemble/screens/widgets.yaml

```diff
@@ -736,6 +737,54 @@ View:
                   value: true
                   leadingText: Set to true and requred
 
+        - Card:
+            styles:
+              maxWidth: unset
+              width: unset
+              gap: 10px
+            children:
+              - Markdown:
+                  text: |
+                    ### Image Cropper
+              - ImageCropper:
+                  id: imageCropper
+                  source: https://img.huffingtonpost.com/asset/5ab4d4ac2000007d06eb2c56.jpeg?cache=sih0jwle4e&ops=1910_1000
+                  styles:
+                    strokeColor: red
+                    strokeWidth: 2px
```

Comment:
Can we make this more similar to flutter with a onCropped event? https://studio.ensembleui.com/app/e24402cb-75e2-404c-866c-29e6c3dd7992/screen/HG8G4ALBohmSzVJTmTYP